docs/api.md

```diff
@@ -0,0 +1,1425 @@
+
+# API
+
+- all paths should be specified treating the user's home directory as root
+
+- The url base, in an actualy deployment, will be:
+  - `https://ci.kbase.us` for CI
+  - `https://next.kbase.us` for Next
+  - `https://appdev.kbase.us` for Appdev
+  - `https://kbase.us` for Production
+
+- The base path in all environments is `/services/staging_service`
+
+- All api paths will be suffixed to the url base and base path. For example:
+
+    `https://ci.kbase.us/services/staging_service/file-lifetime`
+    will invoke the
+    `file-lifetime` endpoint, which returns the number of days a file will be retained.
+
+- For local usage (i.e. spinning the service up locally)
+  - the base url is `http://localhost:3000`
+  - the base path is not required
+  - e.g. `http://localhost:3000/file-lifetime`
+
+- For endpoints that require authorization, the `Authorization` header must be supplied,
+  with the value a KBase auth token.
+
+## Test Service
+
+`GET /test-service`
```

Comment:
This current serves as the /status endpoint. Maybe we can have 2 routes point to the same status function?

Reply:
Yeah, we could, and should, but perhaps not in this PR.